src/frontend/Felles/Kalkulator/Samværskalkulator.tsx

```diff
@@ -62,19 +52,79 @@ const IkonKnapp = styled(Button)`
     height: fit-content;
 `;
 
-const Spacer = styled.div`
-    width: calc(100% - 1600px);
+const UkeTittel = styled(Label)`
+    text-decoration: underline;
+    text-decoration-thickness: 0.125rem;
+    text-underline-offset: 0.25rem;
+`;
+
+const Grid = styled.div`
+    display: grid;
+    row-gap: 1rem;
+    column-gap: 1rem;
+    overflow: hidden;
 
-    @media screen and (max-width: 1900px) {
+    // Styler slik at første element på hver rad er beskrivelsestekster
+    // Støtter fra to kolonner ved smal skjerm og opp til fem kolonner ved bred skjerm
+    span {
         display: none;
     }
+
+    @media screen and (max-width: 1409px) {
+        grid-template-columns: repeat(2, 305px);
+        span {
+            display: block;
+        }
+    }
+    @media screen and (min-width: 1410px) and (max-width: 1739px) {
+        grid-template-columns: repeat(3, 305px);
+        span:nth-of-type(2n + 1) {
+            display: block;
+        }
+    }
+    @media screen and (min-width: 1740px) and (max-width: 2059px) {
+        grid-template-columns: repeat(4, 305px);
+        span:nth-of-type(3n + 1) {
+            display: block;
+        }
+    }
+    @media screen and (min-width: 2060px) {
+        grid-template-columns: repeat(5, 305px);
+        span:nth-of-type(4n + 1) {
+            display: block;
+        }
+    }
+
+    // Styler interne borders for gridden
+    .gridItem {
+        position: relative;
+    }
+
+    .gridItem::before,
+    .gridItem::after {
+        content: '';
+        position: absolute;
+        background-color: ${ABorderDivider};
+        z-index: 1;
+    }
+
+    .gridItem::before {
+        inline-size: 2px;
+        block-size: 17rem;
+        inset-block-start: 0;
+        inset-inline-start: -1rem;
+    }
+
+    .gridItem::after {
+        inline-size: 20.075rem;
+        block-size: 2px;
+        inset-inline-start: 0;
+        inset-block-start: -1rem;
```

Comment:
Det er trolig mulig å oppnå "noe tilsvarende" ved å bruke border-left og border-bottom i @media-taggen ovenfor, men ser at man ikke klarer å fjerne border-bottom for de på nederste rad. 

Det kan være litt krevende å forstå hva som skjer her, men det vil uansett bli litt saus å få dette til å fly på en smooth måte